server/db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE videos_recommendation;
+
+CREATE TABLE videos (
+    id VARCHAR(255) PRIMARY KEY,
+    Date VARCHAR(255),
```

Comment:
Postgres has specific date types that are more suitable to use than `VARCHAR(255)`, I'd recommend using one as it will allow you to do things like comparisons, or extracting a month, more easily in the future :)